dvc/config_schema.py

```diff
@@ -223,4 +223,5 @@ class RelPath(str):
         # enabled by default. It's of no use, kept for backward compatibility.
         Optional("parametrization", default=True): Bool
     },
+    "plots": {"html_template": str},
```

Comment:
Should it be just `plots.html`? To avoid overloading the term "template" (also associated with [Vega customizations](https://dvc.org/doc/command-reference/plots#plot-templates)).